.buildkite/src/jobs/Sample/Pipeline.dhall

```diff
@@ -0,0 +1,12 @@
+let Pipeline = ../../Lib/Pipeline.dhall
+let Command = ../../Lib/Command.dhall
+
+in
+
+Pipeline.build
+  Pipeline.Config::{
+    spec = ./Spec.dhall,
+    steps = [
+      Command.Config::{ command = [ "echo \"hello\"" ], label = "Test Echo", key = "hello", target = <Large | Small>.Small }
```

Comment:
You should make an alias for this size union so that you don't have to always use this syntax.